README.md

```diff
@@ -91,4 +143,4 @@ Contact Us
 
 We're a friendly group, so feel free to chat with us in the "Thimble" channel on Mozilla Chat running on [Mattermost](https://about.mattermost.com). To access Mozilla Chat head to [this link]( http://chat.mozillafoundation.org). Note that you will need to create an account first.
 
-You can also download a mobile or desktop client for Mattermost [here](https://about.mattermost.com/download/#mattermostApps).
+You can also download a mobile or desktop client for Mattermost [here](https://about.mattermost.com/download/#mattermostApps).
```

Comment:
I actually don't know hahah.
Nothing got changed in this part really. Not too sure why github thinks I changed it haha.